clusterloader2/pkg/measurement/interface.go

```diff
@@ -34,6 +34,9 @@ type MeasurementConfig struct {
 	Params map[string]interface{}
 	// TemplateProvider provides templated objects.
 	TemplateProvider *config.TemplateProvider
+	//ClusterLoaderConfig instance for accessing flags
```

Comment:
Add a space after //

Reply:
Also, consider removing the comment, ClusterLoaderConfig is a self-documenting name.